core/controllers/base_test.py

```diff
@@ -1138,7 +1138,11 @@ def test_every_method_has_decorator(self):
         self.assertGreater(len(handlers_checked), 0)
 
         for (name, method, handler_is_decorated) in handlers_checked:
-            self.assertTrue(handler_is_decorated)
+            # Since 'BulkEmailWebhookEndpoint' is a class that is not called
+            # from the frontend, but instead from a third party provider via a
+            # webhook, no decorator is required.
```

Comment:
This seems weird actually. Shouldn't we have a decorator that validates that it's coming from the correct place? Otherwise, what stops anyone from just pinging it with random data?

Reply:
We do check it in the controller itself that the list ID in the request is correct and if not, exit from the function.

Reply:
It's not sufficient, I think -- please confirm that all the checks listed here have been done: https://mailchimp.com/developer/transactional/docs/webhooks/ (in the section about "Authenticating webhook requests")

Reply:
Done, added additional authentication.